tests/nat/mcp/client/test_mcp_client_base.py

```
@@ -599,6 +600,195 @@ def test_tool_client_no_parent_client_raises_error(self):
+        from unittest.mock import AsyncMock
+
+        client = MCPStreamableHTTPClient(url="http://localhost:8080/mcp")
+
+        # Mock the streamable_http_client
+        mock_session_id_callback = MagicMock(return_value="mock-session-id")
+        mock_session = AsyncMock()
+        mock_session.initialize = AsyncMock()
+
+        # Create a mock context manager
+        @asynccontextmanager
+        async def mock_streamable_client(*args, **kwargs):
```

Ruff (0.14.14)

Unused function argument: `args`

(ARG001)